packages/larva-css/src/03-utilities/u-border-radius.common.async.scss

```diff
@@ -4,4 +4,20 @@
 
 .lrv-u-border-radius-5 {
 	border-radius: 5px;
-}
\ No newline at end of file
+}
+
+.lrv-u-border-br-radius-5 {
```

Comment:
Can we name it like `lrv-u-border-radius-br-5`?

Reply:
fixed!

Reply:
@yahilmadakiya can this replace what you wrote in AMH?